go.mod

```diff
@@ -42,18 +42,20 @@ require (
 	google.golang.org/grpc v1.67.3
 	k8s.io/api v0.32.3
 	k8s.io/apiextensions-apiserver v0.32.3
-	k8s.io/apimachinery v0.32.3
+	k8s.io/apimachinery v0.33.0
 	k8s.io/apiserver v0.32.3
 	k8s.io/client-go v0.32.3
 	k8s.io/cluster-bootstrap v0.32.3
 	k8s.io/component-base v0.32.3
 	k8s.io/klog/v2 v2.130.1
-	k8s.io/kube-openapi v0.0.0-20241105132330-32ad38e42d3f
+	k8s.io/kube-openapi v0.0.0-20250318190949-c8a335a9a2ff
 	k8s.io/utils v0.0.0-20241104100929-3ea5e8cea738
 	sigs.k8s.io/controller-runtime v0.20.4
 	sigs.k8s.io/yaml v1.4.0
 )
 
+require sigs.k8s.io/randfill v1.0.0
```

Comment:
Why does this end up as a separate `require`?

Reply:
Fixed in https://github.com/kubernetes-sigs/cluster-api/pull/12191

I think this only ended up here because either an IDE or go somehow adds it per default this way (also happened to me). But yes we want this in the block above.